fsspec/implementations/dropboxdrivefs.py

```diff
@@ -0,0 +1,239 @@
+import requests
+import dropbox
+from ..spec import AbstractFileSystem, AbstractBufferedFile
+from ..caching import AllBytes
+
+
+class DropboxDriveFileSystem(AbstractFileSystem):
+    """ Interface dropbox to connect, list and manage files
+    Parameters:
+    ----------
+    token : str
+          Generated key by adding a dropbox app in the user dropbox account.
+          Needs to be done by the user
+
+    """
+
+    def __init__(self, token, *args, **storage_options):
```

Comment:
storage_options become kwargs and are eventually passed to session.get I think - it should say that here. Are there any options that are worth mentioning?

Reply:
Sorry it is probably because of my english but I am not sure to understand your point here...
And also, no, I don't think there are others options used in this implementation.

Reply:
I mean that any extra keyword arguments get stored in the instance, and passed to the `get` call below. This should be documented, as well as any specific options that this might be useful for.

Reply:
The implementation is not using others options... I am just following the signature in spec.py. 
So, I am not sure to understand what to do here. Sorry :/

Reply:
Oh, I think I was getting confused between the kwargs here and the kwargs for the file class. You should just remove `self.kwargs = storage_options` in that case, since this is done by the superclass anyway.